src/main/java/core/basesyntax/strategy/handler/IncrementHandler.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.strategy.handler;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.OperationHandler;
+
+public abstract class IncrementHandler implements OperationHandler {
+    @Override
+    public void apply(Fruit fruit, int quantity) {
+        fruit = Storage.fruits.getOrDefault(fruit.getName(), new Fruit(fruit.getName()));
```

Comment:
Remember to follow the encapsulation principle. Direct access to Storage.fruits violates encapsulation. Consider adding a method in the Storage class to handle the retrieval and updating of fruits.